spec/controllers/admin/emergency_message_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe Admin::EmergencyMessagesController do
+  before :each do
+    @user = User.create!(name: 'hello', admin: true, email: 'lol@biz.info', password: 'irrelevant')
```

Comment:
@cgiHarland you don't need to replace it everywhere once you start using the factory. It would just one incremental change that would lead toward better tests. No need for big refactoring IMHO. @leoapost what do you think?